addons/website/views/website_templates.xml

```diff
@@ -284,7 +284,7 @@
 </template>
 
 <template id="navbar" name="Navbar">
-    <t t-set="_navbar_expand_class" t-value="_navbar_expand_class is None and 'navbar-expand-lg' or ''"/>
+    <t t-set="_navbar_expand_class" t-value="_navbar_expand_class is None and 'navbar-expand-lg' or _navbar_expand_class"/>
```

Comment:
Mmh, indeed, that's an obvious mistake, thank you for noticing it :grimacing: (Un)fortunately, as it does not create any bug (as @rdeodoo explained, it just ends up being a "boolean" variable which is enough for the standard usecase), I think this change has no value in stable, only risks (breaking a custom xpath, reviving a wrong class that will be added on the navbar, etc).

Could you retarget to master and at the same time also fix the other `t-set="_navbar_expand_class"` which has the same issue in the codebase?

Thank you :pray: 

Note: for your custo in stable, you still have to possibility to inherit this template with a xpath and replace that whole t-set with the bug fixed version to solve your issue.